Notoday/HomePage.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  HomePage.swift
+//  Notoday
+//
+//  Created by Jamie Joung on 11/7/22.
+//
+
+import Foundation
+import SwiftUI
+import CoreData
+
+private func styleEmotions(_ value: String) -> Color {
+        switch value {
+            case "Happy":
+                return Color.green
+            case "Satisfied":
+                return Color.orange
+            case "Neutral":
+                return Color.yellow
+            case "Tired":
+                return Color.blue
+            case "Upset":
+                return Color.red
+            default:
+                return Color.purple
+        }
+    }
+
+
+struct NoteCell : View {
+    
+    @State var note: Note
+    
+    let dateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateStyle = .medium
+        return formatter
+    }()
+    
+    var body: some View {
+        return VStack(spacing: 20) {
+            
+                Text(dateFormatter.string(from: note.noteTimestamp!))
+                    .font(.footnote)
+                    
+                Text(note.noteTitle!)
```

Comment:
Here and in the other spots in this file, fail gracefully with `??` like you used below instead of force-unwrapping. That's a bit annoying about how CoreData works. There are some strategies for dealing with that.